common/src/main/java/com/gregtechceu/gtceu/api/data/chemical/material/properties/ToolProperty.java

```diff
@@ -154,6 +154,7 @@ public static Builder of(float harvestSpeed, float attackDamage, int durability,
                     SAW,
                     HARD_HAMMER,
 //                    SOFT_MALLET,
+                    MINING_HAMMER,
                     WRENCH,
                     FILE,
                     CROWBAR,
```

Comment:
personally, I would agree use the HARD_HAMMER for mining rather than a new tool.